bciers/apps/dashboard/middlewares/withAuthorizationDashboard.test.ts

```diff
@@ -0,0 +1,175 @@
+import { NextURL } from "next/dist/server/web/next-url";
+import { NextFetchEvent, NextRequest, NextResponse } from "next/server";
+import { instance, mock, reset, when } from "ts-mockito";
+import middleware from "../middleware";
```

Comment:
Technically this is testing `bciers/apps/dashboard/middleware.ts` though this currently is the only middleware and all the tests are about this so I thought this made more sense. Also I wasn't sure how to just test this function but found examples online of how to test the NextJS middleware stack.